app/src/main/java/br/org/cesar/discordtime/stickysessions/data/remote/interceptor/HttpNetworkInterceptor.kt

```diff
@@ -0,0 +1,36 @@
+package br.org.cesar.discordtime.stickysessions.data.remote.interceptor
+
+import java.io.IOException
+
+import br.org.cesar.discordtime.stickysessions.data.remote.wrapper.INetworkWrapper
+import okhttp3.Interceptor
+import okhttp3.Request
+import okhttp3.Response
+
+class HttpNetworkInterceptor(private val mNetworkWrapper: INetworkWrapper) : Interceptor {
+
+    private var local: Boolean = false
+
+    init {
+        local = false
+    }
+
+    fun setLocal(local: Boolean) {
+        this.local = local
+    }
+
+    @Throws(IOException::class)
+    override fun intercept(chain: Interceptor.Chain): Response {
+        val request = chain.request()
```

Comment:
Suggestion: Use this directly instead of creating a variable to receive the value